Update README.md (#3)
2gis/native-sdk-ios-swift-package: README.md

# PlatformSDK

> Native SDK for working with the map from the 2GIS.

Reviewer: ```suggestion
A native iOS SDK for working with the 2GIS map.
```

## Installation

> To install the SDK, it is necessary to add this Swift Package to your project.

Reviewer: ```suggestion
In order to use the SDK, add a dependency to this Swift Package to your project.
```

Author: А тут не соглашусь. Пункт же называется "Installation" . Соответственно, мы говорим об "how to install", а не "how to use".

Reviewer: По-моему, у нас скорее заголовок неверный. Непонятно, что мы хотим высказать под "установкой" и зачем это нужно пользователю.

Есть потребность использовать пакет SDK, вот и всё.

> About adding package dependencies you can read [here](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app).

Reviewer: ```suggestion
Read [this article](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to learn how one can add a package dependency.
```

Author: А в чем тут принципиальная разница?

Reviewer: Лишнее обращение через you. Также в изначальном предложении неверная структура. Не очень хорошо начинать полное предложение с About.

Использование "learn" уточняет назначение статьи. Иначе не так естественно выглядит текст: нет явности цели чтения "о пакетных зависимостях".

Author: Хз, мне мои варианты также нравились. Поправил.
